crates/rnote-engine/src/store/keytree.rs

```diff
@@ -69,6 +70,18 @@ impl KeyTree {
     pub(crate) fn clear(&mut self) {
         *self = Self::default()
     }
+
+    pub(crate) fn get_tree(&self) -> &rstar::RTree<KeyTreeObject, rstar::DefaultParams> {
```

Comment:
The idea was to encapsulate all needed functionality from the R-Tree. Better would be to add the methods operating on the R-Tree itself here instead of exposing it to the rest of the crate